draft-ietf-rtcweb-jsep.xml

```diff
@@ -4137,40 +4137,6 @@ respectively.  Please review.
 	     sectionFormat="bare"/> and <xref target="RFC4588" section="8.7"
 	     sectionFormat="bare"/> of <xref target="RFC4588"/>.
 
-<!-- [rfced] Sections 5.9 and 5.11:  We had to change "[RFC4588],
```

Comment:
This comment removal doesn't seem like it should be in this PR. The rest LGTM.

Reply:
The comments are issue 45/#915, which I had attempted to close previously, and it had gotten reverted. Can we say this pull request also closes  issue 45/#915?

Reply:
OK, I understand now. I updated the initial description to reference both issues.